parser/src/main/java/com/linkedin/avroutil1/util/ConfigurableAvroSchemaComparator.java

```diff
@@ -84,115 +102,232 @@ private static boolean equals(AvroSchema a, AvroSchema b, SchemaComparisonConfig
       case DOUBLE:
       case STRING:
       case BYTES:
-        return true;
-
-      //named types
+        break;
 
+      // named types
       case ENUM:
-        return ((AvroEnumSchema) a).getFullName().equals(((AvroEnumSchema) b).getFullName()) && (!considerAliases || hasSameAliases((AvroEnumSchema) a, (AvroEnumSchema) b))
-            //list comparison is sensitive to order
-            && ((AvroEnumSchema) a).getSymbols().equals(((AvroEnumSchema) b).getSymbols());
+        AvroEnumSchema enumA = (AvroEnumSchema) a;
+        AvroEnumSchema enumB = (AvroEnumSchema) b;
+
+        // Compare fullName of enum fields in schemaA and schemaB
+        String enumFullNameA = enumA.getFullName();
+        String enumFullNameB = enumB.getFullName();
+        if (!enumFullNameA.equals(enumFullNameB)) {
+          AvroSchemaDifference difference = new AvroSchemaDifference(a.getCodeLocation(), b.getCodeLocation(), AvroSchemaDifferenceType.ENUM_NAME_MISMATCH,
+              String.format("Name for Enum field %s in schemaA does not match with name enum field %s in schemaB",
+                  enumFullNameA, enumFullNameB));
+          differences.add(difference);
+        }
+
+        // Compare aliases of enum fields in schemaA and schemaB
+        if (considerAliases && !hasSameAliases(enumA, enumB)) {
+          AvroSchemaDifference difference = new AvroSchemaDifference(a.getCodeLocation(), b.getCodeLocation(), AvroSchemaDifferenceType.ALIASES_MISMATCH,
+              String.format("Aliases for enum field %s in schemaA does not match with aliases for enum field %s in schemaB",
+                  enumFullNameA, enumFullNameB));
+          differences.add(difference);
+        }
+
+        // Compare symbols of enum fields in schemaA and schemaB
+        if (!enumA.getSymbols().equals(enumB.getSymbols())) {
+          AvroSchemaDifference difference = new AvroSchemaDifference(a.getCodeLocation(), b.getCodeLocation(), AvroSchemaDifferenceType.ENUM_SYMBOL_MISMATCH,
+              String.format("Symbols %s of enum field %s in schemaA does not match with symbols %s of enum field %s in schemaB",
+                            enumA.getSymbols().toString(), enumFullNameA,
+                            enumB.getSymbols().toString(), enumFullNameB));
+          differences.add(difference);
+        }
```

Comment:
we can move these code to a method to make it more clean.
same for comparing other types.

Reply:
`findEnumDifferences()`

Reply:
Good callout, updated this in the recent commit.